README.md

```diff
@@ -43,59 +58,81 @@ After your Lambda@Edge environment is prepared, install the starter kit:
 
 The Kameleoon AWS Lambda@Edge Starter Kit uses and extends the [Kameleoon NodeJS SDK](https://developers.kameleoon.com/feature-management-and-experimentation/web-sdks/nodejs-sdk) to provide experimentation and feature flagging on the edge.
 
-> Note: To run the Kameleoon NodeJS SDK on the edge, you need to provide an `externalClientConfiguration`. This can be accomplished either by referencing a local file or using the supplied `getClientConfiguration` helper function to retrieve your Kameleoon project's client configuration. The `externalClientConfiguration` is a JSON file that encapsulates all of your feature flags and experiments. The Kameleoon NodeJS SDK needs this data to implement and monitor your feature flag deployments and experiments.
-
 ### Initialization
 
-In the `src` folder of the starter kit, you'll find three JavaScript files:
+In the `src` folder of the starter kit, you will find two TypeScript files:
 
-- `src/viewer_request.js` contains sample code that fetches and caches the client configuration, initializes the Kameleoon SDK with this configuration, and modifies the outgoing request by adding the User ID cookie and feature flag value to the header.
-- `src/viewer_response.js` file contains sample code that fetches the User ID cookie from the incoming request and attaches the cookie to the outgoing response.
-- `src/helpers.js` contains some additional platform-specific code that demonstrates common features of the Kameleoon SDK.
+- `src/handler.ts` contains sample code that initializes and caches the Kameleoon SDK and retrieves the feature flag variation.
+- `src/visitorCodeManager.ts` file contains a custom implementation of Kameleoon Visitor Code Manager, which provides an ability to work with Kameleoon `visitorCode` at the edge.
```

Comment:
- `src/visitorCodeManager.ts` contains a custom implementation of the Kameleoon Visitor Code Manager, which allows you to use the Kameleoon `visitorCode` from the edge.

Also, instead of "the edge", can we say "an edge node". I'm not sure if that's exactly what you meant, but it's more specific.

Reply:
I'll leave this one `at the edge` if you don't mind
https://docs.aws.amazon.com/whitepapers/latest/security-at-the-edge/edge-security.html